service/access.go

```diff
@@ -327,15 +347,19 @@ func (a *accessService) updateAccessToken(ctx context.Context, domain, role, pro
 			return nil, fmt.Errorf("jwt.GetExpirationTime() err: %v", err)
 		}
 
-		a.tokenCache.SetWithExpire(key, &accessCacheData{
+		acd := &accessCacheData{
 			token:             at.AccessToken,
 			domain:            domain,
 			role:              role,
 			proxyForPrincipal: proxyForPrincipal,
 			expiresIn:         expiresIn,
 			expiry:            expTime.Unix(),
 			scope:             at.Scope,
-		}, expTime.Sub(expTimeDelta))
+		}
+		a.tokenCache.SetWithExpire(key, acd, expTime.Sub(expTimeDelta))
```

Comment:
Need to minus memory usage of the old data if cache data exists.
https://github.com/AthenZ/k8s-athenz-sia/blob/74f8e1bc3faac487027514dce8494fed4148c3a4/pkg/token/cache.go#L81-L89

Reply:
@WindzCUHK fixed in https://github.com/AthenZ/athenz-client-sidecar/pull/18/commits/9e50a5bb2a23c6f81e9a2291c2632f5c1971730a